src/main/java/com/gluonhq/substrate/util/ios/Deploy.java

```diff
@@ -96,4 +103,49 @@ public static boolean install(String app) throws IOException, InterruptedExcepti
         }
         return false;
     }
+
+    public static void addDebugSymbolInfo(Path appPath, String appName) throws IOException, InterruptedException {
+        Path applicationPath = appPath.resolve(appName + ".app");
+        Path debugSymbolsPath = Path.of(applicationPath.toString() + ".dSYM");
```

Comment:
This looks strange... should it be `Path debugSymbolsPath =  applicationPath.resolve(".dSYM")`?

Reply:
No, that will produce a different result. We need to create `appName.app.dSYM` directory.